### DISCUSSION TITLE
What is latestMeasure.isValid used for?

### DISCUSSION THREAD

**Discussion by apwelsh** [iopool/community#10] [Community development]
Please explain, what causes isValid to return true or false?

If isValid == false, does this mean the measurement no longer relevant (too old)?

**Comment by mehdyouras** [comment]
Hello,

`isValid` value is never updated after the measurement creation. `isValid` is computed on creation and is used to determine if it should be used to trigger a new recommendation or not.

example:
if a measurement is taken while the probe is outside the water -> `isValid` = `false`
